### PR TITLE
Add remote example

### PR DESCRIPTION
**Pull request by sonjaer**
# TL;DR
Add an example of how to use a remote task (task deployed already in a flyte project)

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
See TL;DR

## Tracking Issue
_NA_

## Follow-up issue
_NA_

### CODE REVIEW

File: flytekit-examples-scala/src/main/scala/org/flyte/examples/flytekitscala/WorkflowWithRemoteTask.scala [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+class WorkflowWithRemoteTask
+    extends SdkScalaWorkflow[RemoteSumTaskOutput](
+      SdkScalaType[RemoteSumTaskOutput]
+    ) {
+
+  override def expand(builder: SdkScalaWorkflowBuilder): Unit = {
+    val a = SdkBindingData.ofInteger(10L)
+    val b = SdkBindingData.ofInteger(10L)
```
Comment (sonjaer) [inline flytekit-examples-scala/src/main/scala/org/flyte/examples/flytekitscala/WorkflowWithRemoteTask.scala:33]:
```suggestion
    val b = SdkBindingData.ofInteger(12L)
```

File: README.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ Package and run:
 $ mvn package
 $ scripts/jflyte register workflows \
   -d=development \
-  -p=flytesnacks \
+  -p=spotify-flytekit-examples \
```
Comment (andresgomezfrr) [inline README.md:68]: I think that maybe it is a little strange to put the `spotify` word on the OSS, right?

File: flytekit-examples-scala/src/main/scala/org/flyte/examples/flytekitscala/WorkflowWithRemoteTask.scala [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    extends SdkScalaWorkflow[RemoteSumTaskOutput](
+      SdkScalaType[RemoteSumTaskOutput]
+    ) {
+
+  override def expand(builder: SdkScalaWorkflowBuilder): Unit = {
+    val a = SdkBindingData.ofInteger(10)
+    val b = SdkBindingData.ofInteger(12)
+
+    val c = builder
+      .apply(new RemoteSumTask().create.withInput("a", a).withInput("b", b))
```
Comment (andresgomezfrr) [inline flytekit-examples-scala/src/main/scala/org/flyte/examples/flytekitscala/WorkflowWithRemoteTask.scala:36]: This is also something that you need to update with our last [PR](https://github.com/flyteorg/flytekit-java/pull/175#pullrequestreview-1269060684).

Comment (sonjaer) [reply flytekit-examples-scala/src/main/scala/org/flyte/examples/flytekitscala/WorkflowWithRemoteTask.scala:36]: I can update it but its not merged yet right?

File: flytekit-examples-scala/src/main/scala/org/flyte/examples/flytekitscala/RemoteSumTask.scala [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    a: SdkBindingData[java.lang.Long],
+    b: SdkBindingData[java.lang.Long]
+)
+case class RemoteSumTaskOutput(c: SdkBindingData[java.lang.Long])
+
+class RemoteSumTask {
+  def create: SdkRemoteTask[RemoteSumTaskInput, RemoteSumTaskOutput] = {
+    SdkRemoteTask.create(
+      /* domain= */ null,
+      /* project= */ "spotify-flytekit-examples",
```
Comment (sonjaer) [inline flytekit-examples-scala/src/main/scala/org/flyte/examples/flytekitscala/RemoteSumTask.scala:32]:
```suggestion
      /* project= */ "flytesnacks",
```

File: README.md (hunk 2 above)
Comment (sonjaer) [inline README.md:68]:
```suggestion
  -p=flytesnacks \
```

**Review by andresgomezfrr — APPROVED** [review]

**Pull request merged**